src/gigs-board/components/posts/Post.jsx

```diff
@@ -255,7 +401,7 @@ let grantNotify = Near.view("social.near", "is_write_permission_granted", {
   predecessor_id: nearDevGovGigsContractAccountId,
   key: context.accountId + "/index/notify",
 });
-if (grantNotify === null) {
+if (grantNotify === Loading) {
```

Comment:
Why this change?

Reply:
On the call yesterday was told to change the null to loading when the page is opened.

Reply:
@jefedeoro I believe whoever said that meant roughly the following:

```js
if (grantNotify === null) {
  return <div>Loading...</div>;
}
```